tests/api/Jalasoft.TeamUp.Resumes.API.Tests/UpdateResumeTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Jalasoft.TeamUp.Resumes.API.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using Jalasoft.TeamUp.Resumes.API.Controllers;
+    using Jalasoft.TeamUp.Resumes.Core;
+    using Jalasoft.TeamUp.Resumes.Core.Interfaces;
+    using Jalasoft.TeamUp.Resumes.DAL.Interfaces;
+    using Jalasoft.TeamUp.Resumes.Models;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Moq;
+    using Xunit;
+
+    public class UpdateResumeTests
+    {
+        private readonly Mock<IResumesService> mockResumesService;
+        private readonly DefaultHttpContext mockHttpContext;
+        private readonly PutResume postResumeSkills;
+
+        public UpdateResumeTests()
+        {
+            this.mockResumesService = new Mock<IResumesService>();
+            this.mockHttpContext = new DefaultHttpContext();
+            this.postResumeSkills = new PutResume(this.mockResumesService.Object);
+        }
+
+        [Fact]
+        public void UpdateResume_Returns_OkObjectResult()
```

Comment:
You need to change the name for this test following this section https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices